pandas/tests/arithmetic/test_numeric.py

```diff
@@ -1297,3 +1305,27 @@ def test_dataframe_div_silenced():
     )
     with tm.assert_produces_warning(None):
         pdf1.div(pdf2, fill_value=0)
+
+
+@pytest.mark.parametrize(
+    "data, expected_data", [([0, 1, 2], [0, 2, 4])],
+)
+def test_integer_array_add_list_like(
+    box_pandas_1d_array, box_1d_array, data, expected_data
+):
+    # GH22606 Verify operators with IntegerArray and list-likes
+    arr = array(data, dtype="Int64")
+    container = box_pandas_1d_array(arr)
+    left = np.array(container + box_1d_array(data))
```

Comment:
don't wrap left / right in np.array; this makes this way more confusing. These types so should preserve.

Reply:
Updated it such that we assert using the original types.